specforge/__init__.py

```diff
@@ -1,2 +0,0 @@
-from .core import *
-from .modeling import *
```

Comment:
I think this deletes too much. How do you do something like

```python
from specforge import AutoDraftModelConfig, AutoEagle3DraftModel, OfflineEagle3Model
```

if `__init__.py` is empty?

@FrankLeeeee